### ISSUE TITLE
How start values from -ve to +ve in BarChrt?

### ISSUE THREAD

**Issue by TanveerMM** [PhilJay/MPAndroidChart#5068]
I have created barChart but I want to show barChart values which should be start from -100 to 0 or +20 etc.

**Comment by imankitaman** [comment]
You can look at BarCharts Negative Example for the same

**Comment by TanveerMM** [comment]
Yes, but values start from 0 to -ve. I want to show that if value is -20 so value start from -100 to -80 but it starts from 0 to -20.
